pre_commit/languages/rust.py

```diff
@@ -52,15 +90,57 @@ def _add_dependencies(
         f.truncate()
 
 
+def install_rust_with_toolchain(toolchain: str) -> None:
+    # Check if rustup is already present in PATH
+    rustup_exe = parse_shebang.find_executable('rustup')
+    with tempfile.TemporaryDirectory() as rustup_dir:
+        rustup_env = (
+            ('RUSTUP_HOME', rustup_dir),
+        )
+        with envcontext(rustup_env):
+            if rustup_exe is None:
+                # We did not detect rustup and need to download it first.
+                if sys.platform == 'win32':  # pragma: no cover
```

Comment:
I think this'd be `# pragma: win32 cover` and the else would be `# pragma: win32 no cover`

Reply:
thanks. https://github.com/pre-commit/pre-commit/pull/2534/commits/59abe3875eb893fed407a410125c8910bcc3d2cf